File: 2574_Left_and_Right_Sum_Differences/solution.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* leftRightDifference(int* nums, int n, int* rs) {
    int* r = (int*)malloc(n * sizeof(int));
    *rs = n;
    int t = 0, l = 0;
    for (int i = 0; i < n; i++) t += nums[i];
    for (int i = 0; i < n; i++) {
        int rt = t - l - nums[i];
        r[i] = abs(rt - l);
        l += nums[i];
    }
    return r;
}

File: 2574_Left_and_Right_Sum_Differences/solution.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        t, l, r = sum(nums), 0, []
        for n in nums:
            rs = t - l - n
            r.append(abs(rs - l))
            l += n
        return r